src/Application.h

```diff
@@ -40,12 +40,22 @@ along with RALibretro.  If not, see <http://www.gnu.org/licenses/>.
 #include "Memory.h"
 #include "States.h"
 
+/* Parsed command line arguments */
+struct parsedArgs
```

Comment:
Please use PascalCase for type names

Reply:
✔️ https://github.com/theMK2k/RALibretro/commit/a1208ef0972cce680ba810ea48f70d4c9f1eeff5